templates/bashrc.osh-template

```diff
@@ -9,7 +9,10 @@ export OSH=~/.oh-my-bash
 
 # Set name of the theme to load. Optionally, if you set this to "random"
 # it'll load a random theme each time that oh-my-bash is loaded.
-OSH_THEME="font"
+OSH_THEME="random"
+
+# If you set OSH_THEME to "random", you can ignore themes you don't like.
+OMB_THEME_RANDOM_IGNORED=("powerbash10k" "wanelo")
```

Comment:
By default, the ignored themes for random should be empty, or this can be commented out as other configuration variables.